hack/multikueue-e2e-test.sh

```diff
@@ -57,7 +57,7 @@ function startup {
             MANAGER_KIND_CONFIG="${SOURCE_DIR}/multikueue/manager-cluster.kind.yaml"
         fi
 
-	      echo "Using manager config: $MANAGER_KIND_CONFIG"
+	    echo "Using manager config: $MANAGER_KIND_CONFIG"
```

Comment:
```suggestion
	      echo "Using manager config: $MANAGER_KIND_CONFIG"
```

nit

Reply:
just wonder why you guys think it should intended?

Reply:
This is not a necessary change that breaks the alignment. Look [here](https://github.com/kubernetes-sigs/kueue/blob/467b4ee707286f1f86f4f94cdb681111152bee61/hack/multikueue-e2e-test.sh#L60).

Reply:
ok, I won't argue ;)